molo/core/templates/patterns/basics/article-meta.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags core_tags i18n comments molo_commenting_tags }
```

Comment:
We don't need this template as well this should be defined in the commenting plugin